openpower/configs/linux/skiroot_defconfig

```diff
@@ -57,85 +57,89 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_INET_XFRM_MODE_TUNNEL is not set
 # CONFIG_INET_XFRM_MODE_BEET is not set
 # CONFIG_IPV6 is not set
```

Comment:
We don't actually support IPv6?  Huh.  I'm surprised nobody has complained about that yet.  I thought it was a requirement in some circles these days.